### PR TITLE
Toolbar: `data-toolbar-order` is now inherited

### PR DESCRIPTION
**Pull request by RunDevelopment**
What the title says. More details in #2200.

### CODE REVIEW

File: plugins/toolbar/prism-toolbar.js [hunk 1]
```diff
@@ -81,22 +103,27 @@
...
 		wrapper.appendChild(pre);
 
 		// Setup the toolbar
 		var toolbar = document.createElement('div');
 		toolbar.classList.add('toolbar');
 
-		if (document.body.hasAttribute('data-toolbar-order')) {
-			callbacks = document.body.getAttribute('data-toolbar-order').split(',').map(function(key) {
+		// order callbacks
+		var elementCallbacks;
```
Comment (mAAdhaTTah) [inline plugins/toolbar/prism-toolbar.js]: assign to `callbacks` here then kill the `else` block.

File: plugins/toolbar/prism-toolbar.js [hunk 2]
```diff
@@ -63,6 +63,28 @@
...
+				order = order.trim();
+				if (order.length) {
+					return order.split(/\s*,\s*/g);
+				} else {
+					return [];
+				}
+			}
+			element = element.parentElement;
+		}
+		return undefined;
```
Comment (mAAdhaTTah) [inline plugins/toolbar/prism-toolbar.js]: We don't need to explicitly return `undefined` here. Just kill this `return`.

Comment (RunDevelopment) [reply plugins/toolbar/prism-toolbar.js]: Ah right. JS does that.

**Comment by RunDevelopment** [comment]
Nice. What should we do with #2199? I still think that the `disable` feature might be easier to use that `order` in some cases. But their functionalities really overlap.

**Comment by mAAdhaTTah** [comment]
Honestly, I think we should close it. If the same result can be produced with the current API, it would be better to push users towards that API than have multiple ways of doing the same thing. Plus the associated code size increase.

**Pull request merged**